### PR TITLE
Add new flash

### PR DESCRIPTION
**Pull request by katzj**

### CODE REVIEW

File: app/views/shared/flashes/upstream-webinar-1.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Live webinar: How is Bloomberg supporting the Python community by shifting left? Join us 8/4.
```
Comment (jhan217) [inline app/views/shared/flashes/upstream-webinar-1.html.erb:1]: Worth adding the year to that date?

File: app/views/shared/flashes/upstream-webinar-1.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Live webinar: How is Bloomberg supporting the Python community by shifting left? Join us 8/4. 
+<strong>
+  <a href="https://upstream.live/webinar/supporting-the-python-community-by-shifting-left?utm_source=libraries&utm_medium=referral&utm_campaign=upstream-webinar"
+    target="_blank"
```
Comment (jhan217) [inline app/views/shared/flashes/upstream-webinar-1.html.erb:4]: Do we control the `https://upstream.live` domain or should we add the `rel="noopener noreferrer"`/`rel="external"` tags?

Comment (katzj) [reply app/views/shared/flashes/upstream-webinar-1.html.erb:4]: We do

File: app/views/shared/flashes/upstream-webinar-1.html.erb (hunk 1 above)
Comment (katzj) [reply app/views/shared/flashes/upstream-webinar-1.html.erb:1]: We swap them out before it matters (I just get the content from Amy :)

**Review by jhan217 — APPROVED** [review]

**Pull request merged**